.env.example

```
@@ -43,6 +52,19 @@ FLUENT_AI_API_PREFIX=
 # Key used to authenticate incoming webhook callbacks from fluent-ai
 AI_INBOUND_SERVICE_KEY=generate-a-32-character-secret-key
 
+# AI suggestion tunables (all optional — shown here at their code defaults).
+# Min translated verses in a project before AI suggestions activate.
+AI_ACTIVATION_THRESHOLD_VERSES=500
+# How many upcoming verses to queue when a chapter is first assigned.
+AI_INITIAL_QUEUE_COUNT=3
+# How many verses ahead of the active verse to pre-generate on navigation.
+AI_DEFAULT_LOOKAHEAD=3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_DEFAULT_LOOKAHEAD key should go before the AI_INITIAL_QUEUE_COUNT key

(UnorderedKey)